### ISSUE TITLE
Add "get started" to description

### ISSUE THREAD

**Issue by abitrolly** [nextcloud-snap/nextcloud-snap#1434]
After `snap install nextcloud` I have no idea how to access the server. The `snap info nextcloud` could at the very least include the `snap logs nextcloud.apache` command to get the address of server (which in my case was 127.0.0.1).  Maybe add `nextcloud.help` command to make the snap self-sufficient.

#### OS/snapd/snap version
Ubuntu 20, nextcloud 19.0.1snap2

**Comment by kyrofa** [comment]
I'm sorry this issue has sat for so long. Honestly you're right-- I tried to make the README a helpful document for getting up to speed on things, but there's no reasonable journey from "I just installed this thing on my computer" to "I'm now reading a document in some github repository".

I'm not a huge fan of the 'help' command idea because I think it's similarly not very discoverable, but I'll look into ways to improve the info page.
